Lombiq.NodeJs.Extensions/Docs/JavaScript.md

```diff
@@ -53,6 +53,12 @@ To use the `npm` scripts defined in this project, add any or all of the followin
   }
   ```
 
+You will also need to add the following line to your project file in a `Property Group` to override the default script execution:
```

Comment:
The future time and obligation are unnecessary complications here. 
Also only use backticks for code. Use underscores for key words. So either

```suggestion
Also add the following line to your project file in a _Property Group_ to override the default script execution:
```
or

```suggestion
Also add the following line to your project file in a `<PropertyGroup>` to override the default script execution:
```